manual_jb/content/_toc.yml

```diff
@@ -57,10 +57,11 @@ chapters:
   sections:
     - file: notebooks/create_dataset/create_dataset.ipynb
     - file: notebooks/create_dfr3_object/create_dfr3_object.ipynb
-    - file: notebooks/create_earthquake/create_earthquake.ipynb
+    - file: notebooks/create_earthquake/create_local_hazard.ipynb
```

Comment:
Can you change this back? I think you accidentally updated the create_earthquake line as well

Reply:
Oh yes I noticed the error and changed it back just a few seconds ago :P